central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,110 @@
+	"github.com/stackrox/rox/pkg/branding"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/templates"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `
+	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+	expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders = `
+	StackRox has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+	expectedNoVulnsFoundEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+	expectedNoVulnsFoundEmailTemplateStackroxBranding = `
+	StackRox has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+	timeStr = `today`
+)
+
+type vulnsAndDate struct {
+	WhichVulns string
+	DateStr    string
+}
+
+var _ suite.SetupAllSuite = (*ScheduleTestSuite)(nil)
+var _ suite.TearDownTestSuite = (*ScheduleTestSuite)(nil)
+
+func TestSchedule(t *testing.T) {
+	suite.Run(t, new(ScheduleTestSuite))
+}
+
+type ScheduleTestSuite struct {
+	suite.Suite
+
+	rc                                              *storage.ReportConfiguration
+	envIsolator                                     *envisolator.EnvIsolator
+	expectedVulnReportEmailTemplateRhacsBranding    string
+	expectedVulnReportEmailTemplateStackroxBranding string
+}
+
+func (s *ScheduleTestSuite) SetupSuite() {
+	s.envIsolator = envisolator.NewEnvIsolator(s.T())
+	s.rc = fixtures.GetValidReportConfiguration()
+
+	data := &vulnsAndDate{
+		WhichVulns: "for all vulnerabilities",
+		DateStr:    timeStr,
+	}
+
+	tmpl, err := template.New("VulnsRHACS").Parse(expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders)
+	s.NoError(err)
+	s.expectedVulnReportEmailTemplateRhacsBranding, err = templates.ExecuteToString(tmpl, data)
+	s.NoError(err)
+
+	tmpl, err = template.New("VulnsStackrox").Parse(expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders)
+	s.NoError(err)
+	s.expectedVulnReportEmailTemplateStackroxBranding, err = templates.ExecuteToString(tmpl, data)
+	s.NoError(err)
+}
```

Comment:
Since both `WhichVulns` and `DateStr` are effectively hardcode now, you can simplify the suite quite a bit by not using templates at all.
Just replace

`Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.`

with

`Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report for all vulnerabilities for today.`

I.e. don't use templating for _expected_ messages.

Reply:
Done